src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -92,8 +88,9 @@
   }
 
   public CompletableFuture<Void> updateInvoiceLine(InvoiceLine invoiceLine) {
-    return handlePutRequest(resourceByIdPath(INVOICE_LINES, invoiceLine.getId()),
-      JsonObject.mapFrom(invoiceLine), httpClient, ctx, okapiHeaders, logger);
+    return calculateInvoiceLineTotals(invoiceLine)
```

Comment:
According to jira description:
1. > Whenever an invoiceLine is created, **updated**, or retrieved, we need to (re)calculate `invoiceLine.adjustmentsTotal` and `invoiceLine.total`.

   Current implementation is just to return `204` status code without body e.g. [PUT /invoice/invoice-lines/{id}](https://s3.amazonaws.com/foliodocs/api/mod-invoice/invoice.html#invoice_invoice_lines__id__put). This is default response type for put responses defined by [item-collection.raml](https://github.com/folio-org/raml/blob/raml1.0/rtypes/item-collection.raml#L63). I discussed this with @craigmcnally and we agreed that at this point no need to change this behavior 
2. > These totals are calculated "on the fly" and never persisted.

   The calculation is done before request is sent to storage which means not readonly properties will be persisted as well

Reply:
So given that we've decided to not return the body on PUT (only 204 as Piotr mentions), there's no need to do the calculation on update... only on POST and GET.  Sorry for the confusion guys.

Reply:
Whether the values are persisted or not isn't really all that important - what's important is that the values are calculated upon record retrieval.  If a persisted value is overwritten, so be it.  

It might be slightly clearer if we weren't persisting any values, but since the storage modules are only intended for internal consumption it shouldn't really matter from a practical perspective

Reply:
The only reason I added the calculations before storing is to handle some exception scenarios, like invalid currency. In case of update as we do not return anything, we could skip this step or just perform separately as needed.

Reply:
okay, got it

Reply:
Removed the calculations in PUT for consistency